### PR TITLE
Add "t6 IoT" to Internet of Things section

### PR DESCRIPTION
**Pull request by mathcoll**
Thank you for taking the time to work on a PR for Awesome-Selfhosted!

To ensure your PR is dealt with swiftly please check the following:

- [x] Submit one item per pull request. This eases reviewing and speeds up inclusion.
- [x] Format your submission as follows, where `Demo` and `Clients` are optional.
  Do not add a duplicate `Source code` link if it is the same as the main link.
  Keep the short description under 250 characters and use [sentence case](https://en.wikipedia.org/wiki/Letter_case#Sentence_case)
  for it, even if the project's webpage or readme uses another capitalisation
  such as title case, all caps, small caps or all lowercase.
  `Demo` links should only be used for interactive demos, i.e. not video demonstrations.
  ``- [Name](http://homepage/) - Short description, under 250 characters, sentence case. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``
- [x] Additions that depend on proprietary services outside the user's control must be marked `⚠`.
  ``- [Name](http://homepage/) `⚠` - Short description, under 250 characters, sentence case. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``
- [x] Additions that are not [Free software](https://en.wikipedia.org/wiki/Free_software) must be added to `non-free.md` and have the license set to `⊘ Proprietary`
- [x] Additions are inserted preserving alphabetical order.
- [x] Additions are not already listed at any of [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin), [awesome-analytics](https://github.com/onurakpolat/awesome-analytics), [staticgen.com](https://www.staticgen.com/), [staticsitegenerators.net](https://staticsitegenerators.net/).
- [x] The `Language` tag is the main **server-side** requirement for the software. Don't include frameworks or specific dialects.
- [x] Any license you add is in our [list of licenses](https://github.com/awesome-selfhosted/awesome-selfhosted/blob/master/README.md#list-of-licenses).
- [x] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted/pulls), including closed ones.
- [ ] Any category you are creating has the minimum requirement of 3 items.
  If not, your addition may be inserted into `Misc/Other`.
- [x] Any software project you are adding to the list is actively maintained.
- [x] The pull request title is informative, unlike "Update README.md".
  Suggested titles: "Add aaa to bbb" for adding software aaa to section bbb,
  "Remove aaa from bbb" for removing, "Fix license for aaa", etc.

### CODE REVIEW

**Review by mathcoll — COMMENTED** [review]
Add "t6 IoT" to Internet of Things section

**Review by Rabenherz112 — CHANGES_REQUESTED** [review]
The linked Website (https://api.internetcollaboratif.info/) is not valid / active.

File: README.md [hunk 1]
```diff
@@ -1092,6 +1092,7 @@ A [human resources management system](https://en.wikipedia.org/wiki/Human_resour
 - [openHAB](https://www.openhab.org) - Vendor and technology agnostic open source software for home automation. ([Source Code](https://github.com/openhab/openhab-core)) `EPL-2.0` `Java`
 - [OpenRemote](https://openremote.io) - Open-Source IoT Platform - IoT Asset management, Flow Rules and WHEN-THEN rules, Data visualization, Edge Gateway. ([Demo](https://demo.openremote.io/), [Source Code](https://github.com/openremote/openremote)) `AGPL-3.0` `Java`
 - [SIP Irrigation Control](https://dan-in-ca.github.io/SIP/) - Open source software for sprinkler/irrigation control. ([Source Code](https://github.com/Dan-in-CA/SIP)) `GPL-3.0` `Python`
+- [t6 IoT](https://api.internetcollaboratif.info/) - Open-source IoT Platform to connect physical Objects to timeseries - data oriented and focused, Data Exploration. ([Demo](https://api.internetcollaboratif.info/#signup)), [Source Code](https://github.com/mathcoll/t6)) `GPL-3.0` `Nodejs`
```
Comment (Rabenherz112) [inline README.md:1095]:
```suggestion
- [t6 IoT](https://api.internetcollaboratif.info/) - Open-source IoT Platform to connect physical Objects to timeseries - data oriented and focused, Data Exploration. ([Demo](https://api.internetcollaboratif.info/#signup), [Source Code](https://github.com/mathcoll/t6)) `GPL-3.0` `Nodejs`
```

**Review by Rabenherz112 — CHANGES_REQUESTED** [review]
As far as I can see, you have one too many brackets. Otherwise, it looks good.

**Comment by nodiscc** [comment]
No reply for > a month, closing.

**Pull request closed without merging**